woocommerce/woocommerce-gutenberg-products-block
# Update base/components classnames

Part of #2457.
Fixes #1455.

Updates the class names of all components inside `base/components` to follow the guidelines. It applies the deprecation policy from https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/2457#issuecomment-642032512:
* if a class name was released with WC Core, we list it in the docs as 'deprecated' but don't remove it, in parallel we add the new one so themes can start using the correct ones
* if a class names was not released with WC Core, we simply replace it with the new one and document the change

### How to test the changes in this Pull Request:

I think the only way to test is checking each single block from the list below to verify there are no visual regressions:

- All Reviews, Reviews by Product, Reviews by Category.
- All Products, Filter by Price, Filter by Attribute and Active Filters.
- Cart and Checkout.

If I'm not wrong, these are the only blocks that use components from `base/components`.

I added the `skip-changelog` label because #2691 already has the changelog entry relative to this change.
